### ISSUE TITLE
[Web Bug] - Xpkg build has incorrect flag

### ISSUE THREAD

**Issue by rj-devberg** [crossplane/docs#798]
the flag  `--embed-runtime-image-name=NAME` does not exist on the xpkg cli build command
![image](https://github.com/user-attachments/assets/696f9dc7-c6fd-4cda-8e57-87b10935708c)

running `xpkg build` with the flag results in 

```
crossplane: error: unknown flag --embed-runtime-image-name
```

Should this instead be 
```
--embed-runtime-image
```

As the writing a function in python guide says here:
![image](https://github.com/user-attachments/assets/d8f48435-1d93-4374-8aa2-cabdbb486138)

URL: https://docs.crossplane.io/latest/cli/command-reference/

**Comment by jbw976** [comment]
Good catch @rj-devberg, thanks for reporting this! Do you have any desire to submit a PR to make this update in the docs repo?

The source file location should be https://github.com/crossplane/docs/blob/master/content/master/cli/command-reference.md, and you would also need to make the same change in the applicable versioned directories, e.g. https://github.com/crossplane/docs/tree/master/content/v1.16/cli
